### PR TITLE
CodeCamp  #MMDetection-46

### PR DESCRIPTION
**Pull request by roger-lcc**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: mmdet3d/models/losses/axis_aligned_iou_loss.py
Comment (Xiangxu-0103) [inline mmdet3d/models/losses/axis_aligned_iou_loss.py:38]: Defaults to `mean`.

File: mmdet3d/models/losses/axis_aligned_iou_loss.py
Comment (Xiangxu-0103) [inline mmdet3d/models/losses/axis_aligned_iou_loss.py:42]: Please add typehint for `__init__`

File: mmdet3d/models/losses/axis_aligned_iou_loss.py
Comment (Xiangxu-0103) [inline mmdet3d/models/losses/axis_aligned_iou_loss.py:52]: I think avg_factor should be `float` instead of `int`

File: mmdet3d/models/losses/chamfer_distance.py
Comment (Xiangxu-0103) [inline mmdet3d/models/losses/chamfer_distance.py:113]: remove Optional

File: mmdet3d/models/losses/multibin_loss.py
Comment (Xiangxu-0103) [inline mmdet3d/models/losses/multibin_loss.py:77]: remove Optional

File: mmdet3d/models/losses/axis_aligned_iou_loss.py
Comment (Xiangxu-0103) [inline mmdet3d/models/losses/axis_aligned_iou_loss.py:60]: It seems that weight should be `Tensor`, not support `float`.

File: mmdet3d/models/losses/rotated_iou_loss.py
Comment (Xiangxu-0103) [inline mmdet3d/models/losses/rotated_iou_loss.py:51]: the same

File: mmdet3d/models/losses/chamfer_distance.py
Comment (Xiangxu-0103) [inline mmdet3d/models/losses/chamfer_distance.py:45]:
```suggestion
            - loss_src (Tensor): The min distance
              from source to destination.
            - loss_dst (Tensor): The min distance
              from destination to source.
            - indices1 (Tensor): Index the min distance point
              for each point in source to destination.
            - indices2 (Tensor): Index the min distance point
              for each point in destination to source.
```

File: mmdet3d/models/losses/axis_aligned_iou_loss.py
Comment (Xiangxu-0103) [reply mmdet3d/models/losses/axis_aligned_iou_loss.py:38]: we should better add mark `''` for string value.

**Comment by roger-lcc** [comment]
Sorry for the late reply. @ @Xiangxu-0103

File: mmdet3d/models/losses/multibin_loss.py
Comment (Xiangxu-0103) [inline mmdet3d/models/losses/multibin_loss.py:27]:
```suggestion
            Defaults to 4.
```

File: mmdet3d/models/losses/multibin_loss.py
Comment (Xiangxu-0103) [inline mmdet3d/models/losses/multibin_loss.py:99]:
```suggestion
        Returns:
```

File: mmdet3d/models/losses/multibin_loss.py
Comment (Xiangxu-0103) [inline mmdet3d/models/losses/multibin_loss.py:29]:
```suggestion
    Returns:
```

File: mmdet3d/models/losses/rotated_iou_loss.py
Comment (Xiangxu-0103) [inline mmdet3d/models/losses/rotated_iou_loss.py:18]: Please add a blank above.

File: mmdet3d/models/losses/rotated_iou_loss.py
Comment (Xiangxu-0103) [inline mmdet3d/models/losses/rotated_iou_loss.py:23]: Please add a blank above.

**Comment by Xiangxu-0103** [comment]
Just need to fix minor comments.

**Comment by roger-lcc** [comment]
:)
Thanks! @Xiangxu-0103

**Review by Xiangxu-0103 — APPROVED** [review]

**Comment by roger-lcc** [comment]
@Xiangxu-0103 
Hi, is there anything else I should do? Or, just waiting for the maintainer's  approval.

**Review by JingweiZhang12 — APPROVED** [review]

**Comment by Xiangxu-0103** [comment]
@roger-lcc Please merge the latest code and fix the lint & UT problem.

**Pull request merged**
